### ISSUE TITLE
Version 2.3.5 is not available on pip

### ISSUE THREAD

**Issue by ollietreend** [danilop/yas3fs#144]
Hi,

Please can we have version 2.3.5 made available on pip?

Currently [pip has the latest version as 2.3.2](https://pypi.python.org/pypi/yas3fs):

```
$ pip search yas3fs
yas3fs (2.3.2)  - YAS3FS (Yet Another S3-backed File System) is a Filesystem in Userspace (FUSE) interface to Amazon S3.
  INSTALLED: 2.3.2 (latest)
```

2.3.5 adds support for sigv4, which I need in order to use yas3fs with one of the newer AWS regions.

As an aside: it'd be great to get a new release of yas3fs since there are 14 changes in master since 2.3.5. But I'll settle for having 2.3.5 available in pip for now. 😄

Thanks

**Comment by Liath** [comment]
We'll need @danilop for this. Seems like it would be rude to ask PyPi support to add peeps as maintainers.

**Comment by bitsofinfo** [comment]
agreed can @danilop please grant all the contributors on this project the proper access to publish PyPi releases?

**Comment by bitsofinfo** [comment]
@danilop ?

**Comment by danilop** [comment]
I'd like to see #147 merged before updating PyPI...

**Comment by fire** [comment]
Should be able to force install using sudo pip install --upgrade git+https://github.com/danilop/yas3fs.git . You may also want to force a particular commit.

**Comment by Liath** [comment]
@danilop ![Your wish is my command](http://3.bp.blogspot.com/-JmnL_o45PCE/UCn4__GMZoI/AAAAAAAAAPU/0YiLlHKrW94/s150/G%25C3%25AAnio%2Bda%2BLampada.gif)

**Comment by bitsofinfo** [comment]
#124

**Comment by rdewaele** [comment]
@danilop Do you have any updates on this? #147 appears to be merged, but 2.3.2 still on PyPI. We are also hitting #104 and would like to update.

Meanwhile I have started testing a workaround, which involves explicitly setting a cache directory upon mount, and a daily cronjob to find old files and delete them: `@daily find /path/to/dedicated/cache/dir -type f -not -atime -14 -exec rm '{}' \;`

( We like to have it KISS on this machine, and this seemed the easiest to 'undo' when 2.3.5 or later lands. :) )

**Comment by danilop** [comment]
I just uploaded version 2.3.5 on PyPI. I also updated the _version.py file. Let me know if that works. Thank you all for your patience.
